dotcom-rendering/src/server/dev-index.html

```diff
@@ -309,6 +309,40 @@ <h2>Test Articles By Element (Missing)</h2>
                 },
             ];
 
+			const urls = new Set();
+
+			const checkClipboard = (e) => {
+				if (document.visibilityState !== "visible") return;
+				if (!navigator?.clipboard) return;
+				if (!document.hasFocus()) return;
+
+				navigator.clipboard.readText().then(text => {
+					const guurl = ["https://www.theguardian.com/", "https://m.code.dev-theguardian.com/", "http://localhost:9000/"].find(base => text.startWith(base));
+					if (text.startsWith(guurl)) {
```

Comment:
This could just be `if (guurl)`? I think it still works either way, just a redundant .startsWith!

Reply:
yes guurl

Reply:
```suggestion
					if (guurl) {
```